src/graph/GraphViewServer.ts

```diff
@@ -373,7 +377,7 @@ export class GraphViewServer extends EventEmitter {
       socketError = err;
     }
 
-    return new Promise<[{}[]]>((resolve, reject) => {
+    return new Promise<[Object[]]>((resolve, reject) => {
```

Comment:
nit: Again, I don't care if you use `any[]` or `Object[]`, but I think this should match the return type of its function